lib/diagnostic.rb

```diff
@@ -22,8 +30,9 @@
 # Finally, assign the modified Person to `Answer.dave` below.
 
 ##
-# your answers here
-Answer.dave = nil
+new_person = Person.new('Dave', 32, 'Ohio')
+new_person[:location] = 'Somerville'
```

Comment:
Almost. You would do `new_person.location` since we would have `location` as a writable attr.